ui_framework/components/info_button/_index.scss

```diff
@@ -0,0 +1 @@
+@import "info_button";
```

Comment:
In practice, the color will be the same blue as links. I have done as you suggested, but given the value for the blue color.